src/renderers/WebGLRenderer.js

```diff
@@ -55,6 +55,10 @@ THREE.WebGLRenderer = function ( parameters ) {
 	this.gammaInput = false;
 	this.gammaOutput = false;
 
+	// physical lights
+
+	this.physicalLights = false;
```

Comment:
I think I would call it `physicalLighting` instead. Or maybe `physicalyCorrect`?